src/main/java/games/strategy/util/MD5Crypt.java

```diff
@@ -18,10 +18,11 @@
  * - Added built-in java.security.MessageDigest (MD5) support
  * - Code cleanup
  * <br>
- * <br>
- * TODO Use SHA512(fast) or BCrypt(secure) in the future instead
- * this may be kept for backwards compatibility
+ *
+ * @deprecated Use SHA512(fast) or BCrypt(secure) in the future instead
+ * (kept for backwards compatibility)
```

Comment:
Checkstyle: "Line continuation have incorrect indentation level, expected level should be 4."